create_reference_tsv.py

```diff
@@ -0,0 +1,48 @@
+import os
+import sys
+import re
+inFile = sys.argv[1]
+outFile = sys.argv[2]
+
+sampleDict = {}
+outList = []
+
+with open(inFile, 'r') as f:
+	lines = f.readlines()
+k = 0
+for i in lines:
+	pattern = r'^(.*)_(S\d+)_L(\d+)_(R\d)_001.*$'
+	bnm = re.match(pattern, os.path.basename(i))
+	#TODO: 5th field in tsv might later on be changed from an int to a string containing "[FCID].[laneno].[ssheet_idx]".
```

Comment:
I think this TODO would be clearer if it's stated in the beginning that the TODO is about also fetching the flowcell ID.